microsoftgraph/microsoft-graph-explorer-v2
# Feat/copy snippet

## Overview

Adds copy functionality for snippets.

### Demo

![image](https://user-images.githubusercontent.com/12011447/69705137-fe77e380-1105-11ea-9372-b39d1a9bc442.png)

### Notes

N/A

## Testing Instructions

* Open GE
* Click on the `snippets` tab
* Click on the copy icon
* Paste in an open file
* Observe that the copied code snippet is pasted

Closes #191
